#!/usr/bin/env python
PACKAGE = "autolabor_pro1_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("left_wheel", int_t, 0, "The number of rotations of the left wheel in a PID control cycle", 0, -32, 32)
gen.add("right_wheel", int_t, 0, "The number of rotations of the right wheel in a PID control cycle", 0, -32, 32)
gen.add("run_flag",   bool_t,   0, "start to run the program",  True)

exit(gen.generate(PACKAGE, "sim_autolabor_driver", "SimAutolaborDriver"))
